modules/bluima_opennlp/src/main/java/ch/epfl/bbp/uima/ae/TokenAnnotator.java

```diff
@@ -55,8 +58,8 @@
      */
     public static final String COMPONENT_ID = "de.julielab.jules.OpenNLPTokenizer";
 
-    @ConfigurationParameter(name = BlueUima.PARAM_MODEL_FILE)
-    private String modelFile;
+    @ConfigurationParameter(name = BlueUima.PARAM_MODEL)
```

Comment:
I can revert it now*, but since the value need editing in any case (from full path to package name) wouldn't it better to use a different name?

If a pipeline mention a non-existing config parameter, will an error or warning be emitted? If that's not the case I could implement that. This way, by changing the parameter name from `PARAM_MODEL_FILE` to `PARAM_MODEL` will make it easy to spot such change in the pipelines. What do you think?

* I prefer doing it now than later.